test/microphysics2M_tests.jl

```diff
@@ -729,27 +734,32 @@ function test_microphysics2M(FT)
                 0,
                 D_max_limited,
             )[1] / ρₐ
-        TT.@test qD ≈ qᵣ atol = FT(1e-6)
-        TT.@test qx ≈ qᵣ atol = FT(1e-6)
-        TT.@test qD_bounded ≈ qᵣ atol = FT(1e-6)
-        TT.@test qD_bounded_limited ≈ qᵣ atol = FT(1e-6)
-
-        # The mass integrals don't work with limiters
-        TT.@test qx_lim ≈ qᵣ atol = FT(1e-6)
-        TT.@test qD_lim ≈ qx_lim atol = FT(1e-6)
+        TT.@test qD ≈ qᵣ rtol = FT(1e-2)
+        TT.@test qx ≈ qᵣ rtol = FT(1e-2)
+        TT.@test qx_lim ≈ qᵣ rtol = FT(1e-2)
+        TT.@test qD_lim ≈ qx_lim rtol = FT(1e-2)
+        if FT == Float64
+            TT.@test qD_bounded ≈ qᵣ rtol = FT(1e-11)
+            TT.@test qD_bounded_limited ≈ qᵣ rtol = FT(1e-11)
+        else
+            TT.@test qD_bounded ≈ qᵣ rtol = FT(1e-4)
+            TT.@test qD_bounded_limited ≈ qᵣ rtol = FT(1e-4)
+        end
     end
 
     TT.@testset "2M_microphysics - Seifert and Beheng 2006 cloud distribution sanity checks" begin
 
         # example number concentration and specific humidity
-        Nₗ = FT(1e9)
-        qₗ = FT(1e-3)
+        Nₗ = FT(1e3 * 1e6) # 1000 1/cm3
+        qₗ = FT(1e-3)      # 1 g/kg
 
         # air and liquid water densities in μg/m3
         ρₐ = FT(1.2)
         ρₗ = SB2006.pdf_r.ρw
 
-        # distribution parameters for cloud, units: [Bc] = 1/μg, [Ac] = 1/m3 1/μg3
+        # distribution parameters for cloud
+        # units: [Bc] = 1/μg, [Ac] = 1/m3 1/μg3
+        # units: [Ec] = 1/mm3, [Cc ] = (1/mm3)^4
```

Comment:
Based on line 774 (below), it seems like unit of Ec must be be 1/mm^\psi_c and unit of Cc also needs to be a function of \phi_c